.github/workflows/install-testing.yml

```diff
@@ -0,0 +1,86 @@
+name: Installation Tests
+
+on:
+  push:
+    branches:
+      - trunk
+    # Always test the workflow after it's updated.
+    paths:
+      - '.github/workflows/install-testing.yml'
+  pull_request:
+    # Always test the workflow when changes are suggested.
+    paths:
+      - '.github/workflows/install-testing.yml'
+  workflow_dispatch:
+    inputs:
+      new-version:
+        description: 'The version to test installing.'
+        type: string
+        default: 'latest'
+
+# Cancels all previous workflow runs for pull requests that have not completed.
+concurrency:
```

Comment:
This prevents multiple workflows from running simultaneously on the same PR branch, or for the exact same commit SHA.